cloud-storage/xml-api/cmdline-sample/src/main/java/StorageSample.java

```diff
@@ -0,0 +1,99 @@
+//[START all]
```

Comment:
I think we should have a coding standard and the current consensus is to use the sun's one, according to:
https://github.com/GoogleCloudPlatform/Template/wiki/style.html

What do you think?